.gitattributes

```diff
@@ -1 +1,2 @@
 *.model filter=lfs diff=lfs merge=lfs -text
+models/*.model filter=lfs diff=lfs merge=lfs -text
```

Comment:
Thanks for the catch.  Looks like the first line is nonsense, and should be removed.  If anything, it should have been:

    **/*.model ...

per https://git-scm.com/docs/gitignore#_pattern_format